completion/bash/_phpbrew

```diff
@@ -6660,6 +6660,270 @@ local argument_min_length=0
         fi
     fi
 };
+___phpbrew_complete_system ()
```

Comment:
I'm not too familiar with the phpbrew codebase so why is this needed at all? Are some commands handled in bash instead of calling the PHP app?

Reply:
> I'm not too familiar with the phpbrew codebase so why is this needed at all?

This specific file is needed for bach completion, you can ignore these changes.

> Are some commands handled in bash instead of calling the PHP app?

Yes, it's a necessity when it comes to managing environment variables within the current shell session. This whole new feature is mostly implemented in Fish and then ported to Bash. See the changes in the `shell/` directory.